mars_api/models.py

```diff
@@ -46,9 +45,10 @@ class Player(models.Model):
     def __str__(self):
         return f"{self.nickname}"
 
+
 class Game(models.Model):
     players = models.ManyToManyField(Player, through="PlayerScore")
-    date = models.DateTimeField()
+    date = models.DateTimeField(default=timezone.now())
```

Comment:
Any reason you added `default` instead of `auto_now` (or `auto_now_add`, don't remember specifically)?

Reply:
Good question: `auto_now_add` does not allow to set the time manually (ignores the provided date). `auto_now` sets the field to _now_ every time the object is saved (after every modification). So `default` is the best option if we want to upload a game from the past.